addons/l10n_fr_pos_cert/static/src/js/pos.js

```diff
@@ -23,38 +23,45 @@ models.PosModel = models.PosModel.extend({
         return false;
       }
       return _.contains(french_countries, this.company.country.code);
-    },
+    }
     disallowLineQuantityChange() {
-        let result = _super_posmodel.disallowLineQuantityChange.bind(this)();
+        let result = super.disallowLineQuantityChange(...arguments);
         return this.is_french_country() || result;
     }
-});
+    cashierHasPriceControlRights() {
```

Comment:
This is the replacement of removing NumpadWidget.js. It's used in computing `get hasPriceControlRights` in the main NumpadWidget component.

Reply:
ok